docs/01-welcome-to-civicactions/training/git-ssh.md

```diff
@@ -22,9 +22,9 @@ Host *.civicactions.net
 Port 940
 ```
 
-## Gitlab and SSH
+## GitLaband SSH
```

Comment:
I think we need the space in here :)

Reply:
Heh - I think I read that upside down in my review. @alanna-casey please update.